test/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+if (WIN32)
+    list(APPEND ADDITIONAL_INCLUDE_DIRS ENV INCLUDE ENV CPATH)
```

Comment:
Here we have a kind of undocumented convention that internal (not intended for users) variables are named in lower case starting with `_onedpl_` prefix, please consider renaming

BTW, I would agree on getting rid of this `if-else` and listing all the paths right in `find_path`

Reply:
It seems the majority of internal variables do not add `_onedpl_` prefix now. I replaced a variable with a name without this prefix.